src/test/java/org/broadinstitute/consent/http/mail/freemarker/FreeMarkerTemplateHelperTest.java

```diff
@@ -175,6 +175,35 @@ void testGetDatasetSubmittedTemplate() throws Exception {
     assertFalse(templateString.contains("${"));
   }
 
+  @Test
+  void testGetDaaRequestTemplate() throws Exception {
+    Writer template = helper.getDaaRequestTemplate("signingOfficialUserName", "userName",
+        "daaName",
+        "serverUrl");
```

Comment:
One thing I like to do that ensures that you're testing what you think you're testing is to abstract these values into random string variables, then checking to see if those variables exist as opposed to testing the same string value all the time.